server/flaskr/database/database.py

```
@@ -0,0 +1,53 @@
+            items = cur.fetchall()
+
+        return items
+    
+    @classmethod
+    def write_query(cls, query, params=None):
+        with Database.get_db_access() as conn:
+            cur = conn.cursor()
+            cur.execute(query, params)
+
+    @classmethod
+    def run_sql_file(cls, filename: str):
```

Ruff (0.15.20)

[warning] Missing return type annotation for classmethod `run_sql_file`

Add return type annotation: `None`

(ANN206)